### PR TITLE
created Product Types view

### PR DESCRIPTION
**Pull request by JamsDuke**
## Pull Request Description

Include the applicable information from the list below here 
1. View model that loads after you click on the "Product Types" link in the nav bar. (Currently throws an error because it cannot display null"
## Steps to Test

Include the applicable information from the list below here
1. Open Products folder and locate Types.cshtml
2. Review code.  View does not currently load, pending fixes to ProductsController
## Link to Feature Ticket

Provide the issue number that this pull request fixes
Example: #40

### CODE REVIEW

**Review by delaineackerman — COMMENTED** [review]
LGTM

File: Views/Products/Types.cshtml [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 
-<div class="col-sm-4">
-  
+@{
+  ViewData["Title"] = "Product Types";
+}
+<div class="container">
+  @foreach (var productType in Model.ProductTypes)
+  {
+    <div class="row top-buffer">
```
Comment (delaineackerman) [inline Views/Products/Types.cshtml]: Put this above where the container is

**Comment by HubDub** [comment]
Looks good.

**Review by thrashMoneyVillainare — COMMENTED** [review]
LGTM

**Review by Jgregnier — COMMENTED** [review]
Pinneapples

**Review by csmalley89 — APPROVED** [review]
🐩 😺 💩

**Pull request merged**